### DISCUSSION TITLE
Image rotates when trying to resize using ImageSharp

### DISCUSSION THREAD

**Discussion by srayan** [SixLabors/ImageSharp#2685] [Q&A] [answered]
We have an image here - https://img-s-msn-com.akamaized.net/tenant/amp/entityid/AA1aSetr.img
Original image -> [AA1aSetr.zip](https://github.com/SixLabors/ImageSharp/files/14502244/AA1aSetr.zip)

We have a service where we can resize this image by passing height/width params
example - https://img-s-msn-com.akamaized.net/tenant/amp/entityid/AA1aSetr.img?w=500

When we try to resize it (by passing that w=500 query param), the image is getting auto rotated (90 degrees to the left).

The images are landscape, but the EXIF data says they should be displayed rotated 90 degrees, so in portrait orientation.
It looks like our resizing (using ImageSharp) isn't respecting the EXIF orientation/rotation property, so when the image is resized (through ImageSharp) it is displayed in landscape.

Exif data (pulled from Adobe Photoshop).
[exifdata.txt](https://github.com/SixLabors/ImageSharp/files/14499903/exifdata.txt)

Exif data (pulled from Gimp).
![exif_Gimp](https://github.com/SixLabors/ImageSharp/assets/6014069/0b444116-3ac9-4ab4-9bf5-f62c70830eaa)

Any idea why this is happening and how to fix it? Please let me know if you want me to file this as a bug.

**Comment by tiesont** [comment, marked as answer]
Possibly related: #1613 and #2644

**Reply by srayan** [reply]
`Image.Mutate(i => i.AutoOrient());` fixed the issue

**Reply by ozerarinc** [reply]
> `Image.Mutate(i => i.AutoOrient());` fixed the issue

thank you for resolve problem, i fix it problem in my code

**Reply by lindexi** [reply]
Awesome, thank you.
